src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/DefaultComesLastCheck.java

```diff
@@ -28,20 +28,56 @@
 
 /**
  * <p>
- * Check that the {@code default} is after all the {@code case}s
- * in a {@code switch} statement.
+ * Check that the {@code default} is after all the {@code case} in a {@code switch} statement.
```

Comment:
Why remove the `s` from `{@code case}s`?

Reply:
`XdocsJavaDocsTest` fails with
```
Expected:
all the {@code case} s
Actual: 
all the {@code case}s
```
Do you know where the extra space might come from?

Reply:
Yes.
https://github.com/checkstyle/checkstyle/blob/master/src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsJavaDocsTest.java#L366-L380

Basically the tool just sees the visible text and not the whitespaces. So it has to use it's brain to estimate where a space should be and should not be.
In this case, it thinks `s` is another word and is like "There should be a space splitting these too apart".

Reply:
Let's move this to separate issue, and proceed with the javadoc sync